.gitignore

```diff
@@ -231,6 +231,7 @@ numpy/core/src/umath/loops_unary_complex.dispatch.c
 # npysort module
 numpy/core/src/npysort/x86-qsort.dispatch.c
 numpy/core/src/npysort/x86-qsort.dispatch.*.cpp
+numpy/core/src/npysort/simd_qsort*.cpp
```

Comment:
I think this is too general, it picks up these files:
```
numpy/core/src/npysort/simd_qsort_16bit.dispatch.cpp
numpy/core/src/npysort/simd_qsort.dispatch.cpp
```

@seiko2plus Shouldn't the generated dispatch files be in the `build` directory?

Reply:
> Shouldn't the generated dispatch files be in the build directory?

yes, unless `inplace` option is enabled